assets/questions/reverseWords.js

```diff
@@ -93,6 +97,14 @@ globalData.questions['reverseWords'] = {  // eslint-disable-line dot-notation
           "last reversed word."
         ].join('')
       ]
+    }, {
+      buggyFunctionName: 'AuxiliaryCode.reverseString',
+      messages: [
+        [
+          "Please re-read the question. You need to reverse each word ",
```

Comment:
I would drop the "please re-read the question" -- it feels a bit aggressive. Maybe "You're reversing the entire string, but you should only be reversing the words, not their ordering. Check the example and try again." or something?